backend/spec/services/api/filterer_spec.rb

```diff
@@ -41,13 +41,46 @@
       end
 
       context "when filtered by full_text param" do
-        let(:filters) { {full_text: "TEST", language: :en} }
-        let!(:correct_project) { create :project, name_en: "TEST" }
-        let!(:different_language_project) { create :project, name_es: "TEST" }
-        let!(:different_text_project) { create :project, name_en: "DIFFERENT" }
+        let(:filters) { {full_text: "banana", language: :en} }
+        let!(:correct_project_en) { create :project, name_en: "yellow banana", language: :en }
+        let!(:correct_project_es) {
+          create :project, name_es: "plátano amarillo", name_en: "yellow banana", language: :es
```

Comment:
You removed one use case from test with this change -- It should verify that language of filter matters. It means that `yellow banana` gets ignored if it is saved at different language (for example `name_es`) and searching is done at another language (for example `en`). Right now this test verifies that `yellow banana` needs to be there, but relation between which language filter uses and at which attribute it is saved at is lost.

Reply:
OK, I'll restore that